SygicUIKit/Classes/Views/SpeedSigns/SYUISpeedControlView.swift

```diff
@@ -78,27 +90,19 @@ public class SYUISpeedControlView: UIView {
     // MARK: - Private Methods
     
     private func initDefaults() {
-        setupCurrentSpeedView()
-        setupSpeedLimit(speedLimitView: speedLimitView)
-        setupSpeedLimit(speedLimitView: speedLimitAmericaView)
+        addSubview(controlsStackView)
```

Comment:
asi to je jedno ale preferoval by som keby sa najskor nastavilo translatesAutoresizingMaskIntoConstraints na false az potom sa views pridavali do view hierarchy. neviem teda ci by mohla nastat situacia ze medzi tym ako sa vypne translate to autolayout engin stihne nalayoutovat posvojom ale je to tak istejsie